src/config.cpp

```diff
@@ -344,10 +344,6 @@ void Config::validate() {
         std::cerr << "grpc_bind_address has invalid format: proper hostname or IP address expected." << std::endl;
         exit(EX_USAGE);
     }
-    if (result->count("rest_port") && ((this->restPort() > MAX_PORT_NUMBER) || (this->restPort() < 0))) {
```

Comment:
Why remove this ?

Reply:
This is duplicated. The same check exists above. I found this via coverage, this if statement was never reached since first check always exits.

Reply:
Good catch.